NewForce-Cohort-7/tabloidmvc-mollasses-gang
# Mc29 register

[Description](https://trello.com/c/GlojO7B7/29-29-register)
Added the feature that an unregistered user may register and sign up for the website. 

**Testing Instructions**

git add and git commit whatever you're working on on your branch
git checkout main
git fetch --a
git checkout mc29Register
If you don't see the most recent version of my files, git pull origin
If you haven't created the Tabloid MVC database, please do so.
Use this to fill out tables and columns: [SQL data] (https://github.com/NewForce-Cohort-7/tabloidmvc-mollasses-
gang/blob/main/SQL/01_TabloidMVC_Create_DB.sql)
Use this to seed the database: [SQL Seed Data] (https://github.com/NewForce-Cohort-7/tabloidmvc-mollasses-gang/blob/main/SQL/02_TabloidMVC_Seed_Data.sql)
Click the execute solid green button to create database


SELECT * FROM UserProfile  it will display all current users in the database and their details.
Click the execute solid green button to view the database
You will see Id |DisplayName |FirstName| LastName| DisplayName | Email |CreateDateTime| ImageLocation| UserTypeId





Click the execute with debugger solid green button to launch the app

If you are logged in please direct attention and select the log out option from the nav bar. 
If logged out correctly your nav bar should only contain links for login and register. Please click the register link in the nav bar. 
When you click on register you will be presented will be a form with information to fill out such as |FirstName| LastName| DisplayName | Email |CreateDateTime| ImageLocation| UserTypeId|. 
Please enter all information then click the create button at the bottom of the form.
After clicking create you will be redirected back to list of users that displays Full Name, Display Name, and UserTypeId.

If you wish to check the data base for your new user, please complete the following 
run a new SQL query with the select statement SELECT * FROM UserProfile 
it will display all current users in the database and their details.
Click the execute solid green button to view the database
You will see Id |DisplayName |FirstName| LastName| DisplayName | Email |CreateDateTime| ImageLocation| UserTypeId
and among this list you should see the user that you registered. 



Checklist:
My code follows the style guidelines of this project
I have performed a self-review of my code
I have commented my code, particularly in hard-to-understand areas
My changes generate no new warnings or errors
I have added test instructions that prove my fix is effective or that my feature works
